packages/core/bootstrap/README.md

```diff
@@ -210,7 +210,16 @@ From the moment the subscription is confirmed, the adapter will start receiving
 
 | Required? |                Name                |                                                                             Description                                                                              | Options | Defaults to |
 | :-------: | :--------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-----: | :---------: |
+|           |        `WS_CONNECTION_KEY`         |                                                                                                                                                                      |         |  undefined  |
```

Comment:
I don't know if you want to include this env vars, they don't have any effect yet